model.py

```diff
@@ -20,12 +20,26 @@ class AliasMatch(orm.Document):
 
 
 class Match(orm.Document):
-    fields = [('winner', orm.ObjectIDField(required=True)),
-              ('loser', orm.ObjectIDField(required=True))]
+    fields = [('match_id', orm.IntField(required=True)),
+              ('winner', orm.ObjectIDField(required=True)),
+              ('loser', orm.ObjectIDField(required=True)),
+              ('excluded', orm.BooleanField(required=True))]
 
     def __str__(self):
         return "%s > %s" % (self.winner, self.loser)
 
+    def get_match_id(self):
+        return self.match_id
+
+    def set_match_id(self, id):
+        self.match_id = id
+
+    def is_excluded(self):
+        return self.excluded
+
+    def set_excluded(self, excluded):
+        self.excluded = excluded
+
```

Comment:
not a big deal, but I think in python (as opposed to languages like java) people tend not to write explicit getters/setters unless there is a good reason to; it is generally more readable to just do `match.match_id = id` than `set_match_id(match, id)` (in the latter you have to look up the set_match_id function to see if it is doing something fancier than just setting match.match_id).